coordinates/w128h32.json.example

```diff
@@ -269,6 +269,11 @@
       }
     },
     "runs": {
+      "runs_hits_errors": {
+        "show": false,
+        "compress_digits": false,
+        "spacing" : 3
+      },
```

Comment:
Does 3 look okay if you turn on show, or is it just a dummy value?

Reply:
dummy value for now - that's why it's turned off. Do you know of anyone who is using this size?

Reply:
I don't know anyone personally, but there definitely are users. I'm happy with a dummy value I was just curious